src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -194,7 +194,12 @@
       {:type (if (= t :fs-ter-substitution) :substitution t)
        :pos base-ppos
        :ref (if (= t :fs-ter-substitution)
-              (str pref (subs ref-prot-rest 0 (max 0 (inc (- (count palt) (count pref))))))
+              (let [pref-len (count pref)
+                    palt-len (count palt)
+                    palt-ter-len (inc palt-len)]
+                (if (<= pref-len palt-ter-len)
+                  (str pref (subs ref-prot-rest 0 (max 0 (inc (- palt-len pref-len)))))
```

Comment:
`(max 0 ...)` is nonsense because `(inc (- palt-len pref-len))` is at least 0 under `(<= pref-len palt-ter-len)`.

Reply:
Oh, exactly, I'm kind of too focused on to keep the original code. I'll fix it!